x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -5,10 +5,10 @@
  * 2.0.
  */
 import { encode } from '@kbn/rison';
-import type { FilterItemObj } from '@kbn/security-solution-plugin/public/common/components/filter_group/types';
 import { DEFAULT_DETECTION_PAGE_FILTERS } from '@kbn/security-solution-plugin/common/constants';
 import { formatPageFilterSearchParam } from '@kbn/security-solution-plugin/common/utils/format_page_filter_search_param';
 
+import { FilterControlConfig } from '@kbn/alerts-ui-shared';
```

Comment:
Currently, this test file is skipped because of being flaky but perhaps you can use this to make sure filters are working correctly in Security Solution.

Reply:
Thank you for testing this. I am already working on removing its flakyness so you do not need to worry about it. Thanks for offering.